### ISSUE TITLE
Objects formed by reflection effect.

### ISSUE THREAD

**Issue by XuyangBai** [waymo-research/waymo-open-dataset#387]
Hi, Thanks for sharing this great dataset. For scenes with large glass, there usually exist some LiDAR points behind the glass maybe due to the reflection, and sometimes the points form well-shaped objects. I think such objects could harm the training process as they are not labelled as ground-truth boxes but can not be easily distinguished from ground-truth boxes without the maps or the road structure information. The following are examples:

![519026_lidar_Inked](https://user-images.githubusercontent.com/15166089/141672121-2c7e7a7c-8568-446f-9759-63fd2b18eb57.jpg)

![519026_img3](https://user-images.githubusercontent.com/15166089/141672124-2232b2ef-03f9-409e-8638-b1170d9d2289.PNG)

![519026_lidar1](https://user-images.githubusercontent.com/15166089/141672205-d03a1331-09c2-4f08-8113-db849f8adbfe.PNG)

You can see there are three well-shaped cars behind the building, which I assume is the reflection of cars on the right side of the ego vehicle. And I use the preprocessing tools of mmdetection3d and I think this frame is from the training set, 519th segment, 26th frame.  

The following example is from the validation set, 91st segment, 13th frame.

![1091013_lidar](https://user-images.githubusercontent.com/15166089/141672184-4c1899d9-ab90-40dc-a024-89d71d4f1083.PNG)

![1091013_img3](https://user-images.githubusercontent.com/15166089/141672186-fbc75a82-a8e1-4312-92dd-3d0e1977d028.PNG)

I am not sure how many frames contain such cases, just wondering do you have any suggestions to avoid the potential negative effects of such reflected objects?

Best,
Xuyang

**Comment by peisun1115** [comment]
Detectors are supposed to understand them based on context and lidar point properties (e.g. second return).

**Comment by XuyangBai** [comment]
Hi, Thanks a lot for your reply. It makes sense to me that the detectors have the potential to distinguish the reflected objects from the real objects based on context (i.e. there aren't any ground points for reflected cars). But could you explain more details about the second strategy(lidar point cloud properties)? What is the principle of distinguishing reflected objects using the two returns?  Are the reflected points usually formed by the second return? 

Moreover, what is the recommended way to combine lidar points from first and second returns? In mmdetection3d, I find points from two returns are simply concatenated without adding an indicator value. In this way, I think the network is unable to leverage the information about first and second returns.

Looking forward to your explanation.

Best,
Xuyang
